packages/hub/db/migrations/20221215134519797_reward-proofs.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationBuilder } from 'node-pg-migrate';
+
+const TABLE = 'reward_proofs';
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable(TABLE, {
```

Comment:
Do we need any indexes on this table?

Reply:
Ive added a compound index `payee, reward_program_id`

Reply:
Thanks! Ive added compound index [payee, reward_program_id]